### PR TITLE
Documentation/zenko 1547 update ux images

### PR DESCRIPTION
**Pull request by wabernat**
This PR revisits every process step in the Orbit UI documentation and replaces all outdated screengrabs. We need this because our docs should not tell people to do something they can't do.

This PR fixes #ZENKO-1547

Sorry about the length, but such is the nature of my work.

### CODE REVIEW

File: docs/docsource/installation/install/Install_Zenko.rst [hunk 1]
```diff
@@ -24,7 +24,10 @@ Follow these steps to install Zenko with ingress.
 
 .. _create_options.yaml:
 
-1. Create an options.yaml file in Zenko/kubernetes/ to store deployment
+Final Preparations
```
Comment (JianqinWang) [inline docs/docsource/installation/install/Install_Zenko.rst]: Maybe name this `Configuration Setup`/`Setup Configuration`? Having the first section named Final Preparations could throw people off

File: docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/Setting_up_a_Sandbox_Instance.rst [hunk 2]
```diff
@@ -17,8 +17,12 @@ your data. To set up a Sandbox instance:
 
    |image0|
 
-#. The Welcome dialog displays:
+#. Click **Register with Google**. You must have a Google ID to authenticate
+   against.
```
Comment (JianqinWang) [inline docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/Setting_up_a_Sandbox_Instance.rst]: `authenticate with`?

**Review by JianqinWang — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

File: docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/Setting_up_a_Sandbox_Instance.rst (hunk 2 above)
Comment (wabernat) [reply docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/Setting_up_a_Sandbox_Instance.rst]: changed to: "You must authenticate using a Google ID."

File: docs/docsource/installation/install/Install_Zenko.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/installation/install/Install_Zenko.rst]: Changed to "Configure with options.yaml" 

(The only reason I added these headings was to give the cross-references something to push against...)

**Comment by wabernat** [comment]
@bert-e approve

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e status

**Pull request merged**